public/components/metrics/sidebar/__tests__/sidebar.test.tsx

```diff
@@ -47,10 +60,26 @@ describe('Side Bar Component', () => {
         },
       ],
     });
+    // SavedObjectsActions.getBulk = jest
+    //   .fn()
+    //   .mockResolvedValue({ observabilityObjectList: [{ savedVisualization: sampleSavedMetric }] });
+
+    // coreRefs = mockCoreRefs;
+    // coreRefs.pplService.fetch = jest.fn(() =>
+    //   Promise.resolve({
+    //     data: { DATA_SOURCES: ['datasource1', 'datasource2'] },
+    //     then: () => Promise.resolve(),
+    //   })
+    // );
 
     const wrapper = mount(
       <Provider store={store}>
-        <Sidebar />
+        <Sidebar
+          selectedDataSource={''}
+          setSelectedDataSource={setSelectedDataSource}
+          selectedOTIndex={''}
+          setSelectedOTIndex={setSelectedOTIndex}
+        />
       </Provider>
     );
```

Comment:
We should more test cases such as click for datasource selected, click on metric selected. Covering all sidebar scenarios.

Reply:
will add more, working on it